### PR TITLE
Extend test coverage for tests scenarios with LDAP

### PR DESCRIPTION
**Pull request by jakubschwan**

### CODE REVIEW

File: framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/GenericScenarioImpl.java [hunk 1]
```diff
@@ -127,6 +127,13 @@ public void deploy() {
 
         }
 
+        if (scenarioSettings.getLdapSettings() != null) {
+            scenarioSettings.getAllSettings().stream().forEach((DeploymentSettings deploymentSettings) -> {
+                Map<String, String> envVatiables = deploymentSettings.getEnvVariables();
```
Comment (sutaakar) [inline framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/GenericScenarioImpl.java]: envVatiables -> envVariables

**Comment by jakubschwan** [comment]
jenkins retest this

**Comment by jakubschwan** [comment]
jenkins retest this

**Comment by jakubschwan** [comment]
jenkins retest this

**Comment by jakubschwan** [comment]
retest this please

**Pull request merged**